### ISSUE TITLE
A11y_SQL Azure Data Studio_Import ExtensionModify Columns_Keyboard: On tab key navigation to each selected checkbox the focus on it is not visible.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#12979]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)– Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev1;#BM_SQLAzureDataStudio_Oct2020;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG2.4.7;#Benchmark;#Keyboard;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to the connected server and press enter key, use down key and select any database keep_MDW Right click on/ open the context menu for a database and select Data-tier application wizard from the context menu.
4. Add data in specify input file screen and tab to "next" button and press enter.
5. Press enter on next button.
6. Tab to cell column 3 row1 while the checkbox is selected.

**Actual:**
On navigating to column 3 row 1 checkbox while it is selected the tab focus on it is not visible.
Same is observed for checkbox in column 4 row1 and in row2 column 3 and 4

**Expected:**
On navigating to the checkbox while it is selected the focus should be visible on tab navigation.

**User Impact:**
The keyboard users will not know where the tab navigation is moving .

**Attachment:**
[#12979_A11y_SQL Azure Data Studio_Import ExtensionModify Columns_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/5401662/12979_A11y_SQL.Azure.Data.Studio_Import.ExtensionModify.Columns_Keyboard.zip)

**Comment by fsteffi** [comment]
hi @alanrenmsft has the item "Data-tier application wizard " been removed

GithubTags:#Regressed:02-20-23;

**Comment by fsteffi** [comment]
GithubTags:#Closed;

Version: 1.42.0-insider
Commit: 872095ed849537d6c96373eedcd187b3f707c460
Date: 2023-02-23T06:29:37.626Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.25300

[12979.webm](https://user-images.githubusercontent.com/24580570/220862108-804e4588-07b5-4bcf-b1e9-1cdac3c7e546.webm)
